modules/c++/nitf/unittests/test_image_segment_blank_nm_compression.cpp

```diff
@@ -0,0 +1,331 @@
+#include <cstdlib>
+#include <ctime>
+#include <memory>
+#include <iostream>
+#include <iomanip>
+#include <fstream>
+#include <sstream>
+#include <import/nitf.h>
+#include <import/types.h>
+#include <nitf/ImageSegmentComputer.h>
+#include <nitf/ImageSubheader.hpp>
+#include <nitf/Record.hpp>
+#include <nitf/Writer.hpp>
+#include <nitf/Reader.hpp>
+#include <math/Round.h>
+#include <io/FileInputStream.h>
+#include <io/TempFile.h>
+#include "TestCase.h"
+
+extern "C"{
+   NITF_BOOL nitf_ImageIO_getMaskInfo(nitf_ImageIO *nitf,
```

Comment:
Style: with `*` here and `&` below, they should be on the type-side. (e.g. `nitf_ImageIO* nitf`)